lib/matplotlib/style/core.py

```diff
@@ -45,7 +45,7 @@ def _remove_blacklisted_style_params(d, warn=True):
             if warn:
                 warnings.warn(
                     "Style includes a parameter, '{0}', that is not related "
-                    "to style.  Ignoring".format(key))
+                    "to style.  Ignoring".format(key), stacklevel=2)
```

Comment:
Could be `stacklevel=3` because it's inside a private wrapper.

Reply:
@cldssty did you want to make this change?  Prob fine as a separate commit if you don't feel like rebasing...